cg/services/illumina/post_processing/utils.py

```diff
@@ -14,6 +14,11 @@
 
 LOG = logging.getLogger(__name__)
 
+def sample_is_negative_control_with_reads_in_lane(
+    is_negative_control: bool, metric: IlluminaSampleSequencingMetricsDTO
+) -> bool:
+    """Check if a sample is a negative control and has reads."""
```

Comment:
```suggestion
def sample_is_negative_control_with_reads_in_lane(
    is_negative_control: bool, metric: IlluminaSampleSequencingMetricsDTO
) -> bool:
    """Check if a sample is a negative control and has reads on a lane."""
```

Reply:
I think the docstring just adds confusion in this case 🙈